app/controllers/web/admin/news_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Web::Admin::NewsController < Web::Admin::ApplicationController
+  def index
+    @published_news = Web::NewsDecorator.decorate_collection News.published.order('created_at DESC') #FIXME using another decorator in that file
+    @unpublished_news = Web::NewsDecorator.decorate_collection News.unpublished.order('created_at DESC')
+  end
+
+  def show
+    @news = Web::NewsDecorator.decorate News.find params[:id]#FIXME I'm don't know how to include decorate there into code
```

Comment:
Line is too long. [122/80]<br>Missing space after #.<br>Put a space before an end-of-line comment.